internal_metrics/metrics.yaml

```diff
@@ -0,0 +1,247 @@
+## Notes:
+##  * In most cases, any internal metric measurement should have all dimensions available from the dimensions document available in both source and destination form.
```

Comment:
is this true for internal metric dimensions?

Reply:
removed.